server/web/Dockerfile

```diff
@@ -2,7 +2,9 @@ FROM python:3.6
 
 RUN pip install \
     Flask==0.12.0 \
-    Flask-Bootstrap==3.3.7.1
+    Flask-Bootstrap==3.3.7.1 \
+    Flask-SQLAlchemy==2.2 \
+    Werkzeug==0.14
```

Comment:
`Werkzeug` is unnecessary here since it is a package included in `Flask`. It is technically already installed.